### ISSUE TITLE
Does not generate "Tools" menu option

### ISSUE THREAD

**Issue by jfh232** [MicrosoftDocs/visualstudio-docs#2811]
I believe I followed all the steps, but this is not adding a menu selection to "tools" in my version (2017 15.9.7)  Not sure what is not working.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 99d54da6-38c2-4cc4-9b22-a8f817c83f29
* Version Independent ID: fffed6f2-3b8e-d2c7-e3e3-34ec8bf090d2
* Content: [Hello World extension tutorial - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/extensibility/extensibility-hello-world?view=vs-2017)
* Content Source: [docs/extensibility/extensibility-hello-world.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/extensibility/extensibility-hello-world.md)
* Product: **visual-studio-windows**
* GitHub Login: @gregvanl
* Microsoft Alias: **gregvanl**

**Comment by gregvanl** [comment]
Hi @jfh232 I was able to successfully complete the tutorial with VS 2017 15.9.9.
Could you be looking at the Tools menu in your development instance of VS 2017 instead of the debugger launched Experimental Instance? You should see the words "Experimental Instance" in the title bar of the instance with the new command.

**Comment by jfh232** [comment]
I'm sorry I did not comment again.  I actually did some more testing and found that I was having some problems in the Community version with permissions.  If I ran this as "Administrator", this worked.  Doing a little more digging, as one of the errors I was getting was "missing Key.snk file".  I'm not very similar with private/public signing, but with a little hunting, I was able to

- build a key pair
- Started new template project
- Went into the solution settings and pointed to my new key
- Added a item
- Built
- Worked

So the tutorial just missed either running "Admin" or attaching the key pair to the solution.

Would be nice to add that, as this was somewhat painful to track down.
